components/tabs/tab-panel-mixin.js

```diff
@@ -0,0 +1,65 @@
+import { css } from 'lit-element/lit-element.js';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+
+export const TabPanelMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			noPadding: { type: Boolean, attribute: 'no-padding', reflect: true },
+			role: { type: String, reflect: true },
```

Comment:
I can probably remove this.  Without this, the `role` will not get reflected on the elements as an attribute, but that perhaps does not matter for screen readers.  Currently, we are not using the attribute for CSS selectors or for querying.

Reply:
I've done some testing for this - so for at least VO, if the `tabpanel` role is not reflected to the attribute, the panel is not announced correctly when focus is moved into the panel.

The other thing I noticed for `d2l-tab-internal` was that axe tests would fail unless the `role="tab"` is reflected to the attribute, because it looks for children (of the element with `role="tablist"`) that have the attribute.

Result may vary depending on browser, screen reader, and perhaps role, but based on what I've found here, I'm tempted to suggest that we should always reflect the role.

Reply:
Gotcha -- makes sense. Thanks for investigating.